Packs/RedCanary/Integrations/RedCanary/RedCanary_test.py

```diff
@@ -179,3 +199,24 @@ def test_fetch_multiple_times_with_new_incidents(mocker):
     # only one incidents is being created out of the 2 that were fetched
     assert len(incidents) == 1
     assert last_run["time"] == latest_time_of_occurrence_of_incidents2
+
+
+def test_def_get_full_timeline(mocker):
+    """Unit test
+    Given
+    - raw response of the http request
```

Comment:
what is the difference in between result1 and 2? please explain in the docstring.